src/Managers/ConfigManager.php

```diff
@@ -53,11 +42,23 @@ public function addDefault(ConfigContract $config): void
     public function get(string $class, string $host): ConfigContract
     {
         $config = $this->configs[$class][$host] ?? $this->getDefault($class);
-        assert(is_a($config, $class));
+        assert($config instanceof $class);
 
         return $config;
     }
 
+    /**
+     * @param class-string<ConfigContract> $class
+     */
+    private function getDefault(string $class): ConfigContract
+    {
+        if ($this->existsDefault($class) === false) {
```

Comment:
I would rename `existsDefault` to `defaultExists`.

Make more sense in english

Reply:
ok